ROMFS/px4fmu_common/init.d/10021_3dr_solo

```diff
@@ -0,0 +1,30 @@
+#!nsh
+#
+# @name 3DR DIY Quad
```

Comment:
These are exported in QGC, so please adjust the comments (type should remain the same)